Add three glossary pages related to rate limiting (#35406)
mdn/content: files/en-us/glossary/debounce/index.md

---
title: Debounce
slug: Glossary/Debounce
page-type: glossary-definition
---

{{GlossarySidebar}}

> **Debouncing**, in the context of programming, means to "batch" all operations that were requested in an interval into a single invocation.

Reviewer: Minor reword, what do you think?

```suggestion
**Debouncing**, in the context of programming, means to "batch" all operations requested during a specific interval into a single invocation.
```

> Debouncing is very similar to {{glossary("throttle", "throttling")}}. The key difference is that when invocations happens continuously, throttling ensures that the operation is still performed at a certain maximum rate, while debouncing waits indefinitely until the invocations stop for a certain amount of time.

Reviewer: Alternate wording to consider if you like it:

```suggestion
Debouncing is very similar to {{glossary("throttle", "throttling")}}. The key difference is that throttling enforces a maximum rate on continuous operations, while debouncing waits for invocations to stop for a specific time to consolidate multiple, noisy invocations into one single invocation.
```
